docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -1,35 +1,26 @@
 # How To Upgrade Legacy Map Features
 
-A list of (case sensitive) changes to upgrade legacy TripleA Game XML properties and options
-to the most recent.
+A list of changes to upgrade legacy TripleA Game XML properties and options to the most recent.
  
-When changes are required, all occurrences of the code before the `>>` must be substituted
-with the code after the `>>`. In any code, the "\*" symbol means whatever (the change or deletion is
-to be applied to every given code having whatever character or characters in place of the "\*").
```

Comment:
I assume you assume that the "*" symbol is itself meaningful enough for everyone that there is no need to explain it means something like "zero or more of any characters". If so, I tend not to agree. I believe that a code like
`<triplea minimumVersion="*"/>`:`<triplea minimumVersion="1.9.0.0"/>`
should better have an explanation about the fact that the original "minimumVersion" value can be whatever.

Reply:
We can replace the "*" with "..." if you think it might be confusing. I don't think a reasonable map maker is going to be confused by seeing that XML example.

I'm not sure if the min version should ever have been set to 1.9.0.0 without the build number. I dont' think this is worth spending more time on right now as the min version might be removed from XML

Reply:
I've not tested it recently, but I believe that in the past only the first three digits were accounted (so "1.9.0.0" would be actually the same as writing "1.9.0"). I would be surprised if minimum version 1.9.0.0.x would actually exclude lower 1.9.0.0 versions.
Of course, it should have had a 5th digit, since not all 1.9.0.0 versions are fine, but the problem is that I believe the program was not reading it (I'm not sure, so here I would need a developer to verify and document if there is a digit limit and, if so, what it is).